_posts/2019-08-06-Jolo.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: Jolo
+technology: Java
+image: jolo.jpg
+desc: A jOOQ utility library for object loading
+proj-url: https://www.github.com/picnicsupermarket/jolo
+proj-num: 07
```

Comment:
Octal? :thinking:

Reply:
Come again?

Reply:
Haha, I mean, "do we need that leading zero?"

Reply:
Ah, it's used in the other posts so I assume its a jekyll thing

Reply:
Just tried: if I remove them then that has no impact on the generated code. So push a commit to drop them.